tests/manage/mcg/test_nsfs.py

```diff
@@ -24,6 +25,70 @@ class TestNSFSObjectIntegrity(MCGTest):
 
     """
 
+    @pytest.fixture(autouse=True, scope="class")
+    def modify_endpoint_scc(self, request) -> None:
+        """
+        This fixture modifies the noobaa-endpoint SCC back to the way it was before ODF 4.12.
+        See https://url.corp.redhat.com/5ceb453 for details.
+
+        """
+
+        ocp_scc = ocp.OCP(
+            kind=constants.SCC, namespace=config.ENV_DATA["cluster_namespace"]
+        )
+        nb_endpoint_scc_name = constants.NOOBAA_ENDPOINT_SERVICE_ACCOUNT_NAME
+        nb_endpoint_sa = constants.NOOBAA_ENDPOINT_SERVICE_ACCOUNT
+
+        # Modify the noobaa-endpoint SCC back to the way it was in ODF 4.11
+        json_payload = [
+            {"op": "replace", "path": "/seLinuxContext/type", "value": "MustRunAs"},
+            {"op": "add", "path": "/users/0", "value": f"{nb_endpoint_sa}"},
+        ]
+
+        ocp_scc.patch(
+            resource_name=nb_endpoint_scc_name,
+            params=json_payload,
+            format_type="json",
+        )
+
+        # Verify the changes
+        scc_dict = ocp_scc.get(resource_name=nb_endpoint_scc_name)
+        assert (
+            scc_dict["seLinuxContext"]["type"] == "MustRunAs"
+        ), "Failed to modify the noobaa-db SCC seLinuxContext type"
+        assert (
```

Comment:
Failure here wont be handled by finalizer i guess. may have to handle through exception block. can you verify

Reply:
Fixed  in 3a8419652df6170cb225428170115076036a3621